lib/GLPI/Agent/Task/Inventory/Linux/AntiVirus/Bitdefender.pm

```diff
@@ -0,0 +1,71 @@
+package GLPI::Agent::Task::Inventory::Linux::AntiVirus::Bitdefender;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use UNIVERSAL::require;
+
+use GLPI::Agent::Tools;
+
+sub isEnabled {
+    return canRun('/opt/bitdefender-security-tools/bin/bduitool');
+}
+
+sub doInventory {
+    my (%params) = @_;
+
+    my $inventory = $params{inventory};
+    my $logger    = $params{logger};
+
+    my $antivirus = _getBitdefenderInfo(logger => $logger);
+    if ($antivirus) {
+        $inventory->addEntry(
+            section => 'ANTIVIRUS',
+            entry   => $antivirus
+        );
+
+        $logger->debug2("Added $antivirus->{name}" . ($antivirus->{version} ? " v$antivirus->{version}" : ""))
+            if $logger;
+    }
+}
+
+sub _getBitdefenderInfo {
+    my (%params) =  (
+        command => '/opt/bitdefender-security-tools/bin/bduitool get ps',
+        @_
+    );
+
+    my @output = getAllLines(%params)
+        or return;
+
+    my $av = {
+        "name" => 'Bitdefender Endpoint Security Tools (BEST) for Linux',
+        "manufacturer" => 'Bitdefender',
+        "company" => 'Bitdefender',
+        "uptodate" => 1,
+    };
```

Comment:
Hash values are also wrongly initialized here. And we don't need "MANUFACTURER".

```suggestion
    my $av = {
        NAME     => 'Bitdefender Endpoint Security Tools (BEST) for Linux',
        COMPANY  => 'Bitdefender',
        UPTODATE => 1,
    };
```